Tasbeh/Tasbeh/ViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ViewController.swift
+//  Tasbeh
+//
+//  Created by Rayan Taj on 24/11/2021.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var plusButton: UIButton!
+    @IBOutlet weak var mainView: UIView!
+    
+    @IBOutlet weak var minusButon: UIButton!
+    
+    @IBOutlet weak var tasbehaCounter: UILabel!
+    let deafults = UserDefaults.standard
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let count = deafults.integer(forKey: "count")
+        
+        tasbehaCounter.text = String(count)
+        
+        mainView.layer.cornerRadius = 100
```

Comment:
It looks amazing! I see you're playing with the layer 😎 try out some ideas on the layer styling 
**check this article:**
[https://www.advancedswift.com/corners-borders-shadows/](https://www.advancedswift.com/corners-borders-shadows/
)